2

```diff
@@ -0,0 +1,38 @@
+/**
+ * Copyright 2022 Shift Crypto AG
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export const convertDateToLocaleString = (
+  date: string,
+  language: string
+) => {
+  return new Date(date).toLocaleString(language, {
+    weekday: 'long',
+    year: 'numeric',
+    month: 'long',
+    day: 'numeric',
+    hour: '2-digit',
+    minute: '2-digit',
+  });
+};
+
+export const parseTimeShort = (time: string, lang: string) => {
+  const options: Intl.DateTimeFormatOptions = {
+    year: 'numeric',
+    month: 'numeric',
+    day: 'numeric',
+  } as Intl.DateTimeFormatOptions;
+  return new Date(Date.parse(time)).toLocaleString(lang, options);
+};
```

Comment:
remove? :)

Reply:
I have no idea how this happened, sorry.

Reply:
no problem, maybe quickly check your own PR after pushing, then you'd see such accidental commits.